clang/test/CodeGen/integer-overflow.c

```diff
@@ -70,6 +77,7 @@ void test1(void) {
   // WRAPV: add i8 {{.*}}, 1
```

Comment:
L72 needs a `// CATCH_WRAP: getelementptr i32, ptr`

Actually, since -fsanitize=signed-integer-overflow and -fwrapv -fsanitize=signed-integer-overflow share so many checks. Perhaps share the check prefixes?

```
--check-prefixes=CATCH_UB,CATCH_UB_POINTER
--check-prefixes=CATCH_UB,NOCATCH_UB_POINTER
```

Reply:
How's [1d9cb0a](https://github.com/llvm/llvm-project/pull/82432/commits/1d9cb0aca8985aa1636780b3ff9a863962cc2d57) look?